rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -129,6 +129,18 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 
 		std::vector<std::string> selected_ls(selected.begin(), selected.end());
 		m_emu_settings->SaveSelectedLibraries(selected_ls);
+
+		if (!m_emu_settings->ValidateSettings(false) && QMessageBox::question(this, tr("Remove unused settings?"),
+			tr(
+				"Your config file contains one or more unused entries.\n"
+				"Consider that a removal might render them invalid for other versions of RPCS3.\n"
+				"\n"
+				"Do you wish to let the program remove them for you now?"
+			), QMessageBox::Yes | QMessageBox::No, QMessageBox::No) == QMessageBox::Yes)
```

Comment:
Maybe inform the user about their entry names here because it is important information for answering this question.

Reply:
I'm not sure users should care in the first place when you are requesting expert-level feature.

Reply:
Most people don't even know how some of these settings are called in the config.
And probably even fewer people even use old builds or volatile debug settings.
I'd rather consider just removing them without asking.

Reply:
I don't want my settings removed in a manner I can't control. I'd prefer NOT removing anything even for end users. I don't even see what problem it tries to solve for plain users. If they run some old version, different entries get added again (defaulted), but new entries may get removed without question. This looks like it does more harm than good in many cases.